charts/t8s-cluster/templates/_helpers.tpl

```diff
@@ -30,5 +30,5 @@ sourceRef:
       {{- $hasGPUFlavor = true -}}
     {{- end -}}
   {{- end -}}
-  {{- $hasGPUFlavor -}}
+  {{- $hasGPUFlavor | ternary true "" -}}
```

Comment:
Why is it necessary to add the ternary operator? At first glance, it seems unnecessary as nothing else has changed.

Reply:
`{{ if include "t8s-cluster.hasGPUNodes" ([...]) }}` would also succeed when a `false` is being returned from here, as `"false"`, as a string, as it's non-empty, is a truthy value, therefore we switch the false to `` (empty string), which results in a falsey value